lib/capybara/node/finders.rb

```diff
@@ -151,14 +151,12 @@ def find_link(locator = nil, **options, &optional_filter_block)
       ##
       #
       # Find a button on the page.
-      # This can be any \<input> element of type submit, reset, image, button or it can be a
-      # \<button> element. All buttons can be found by their id, name, Capbyara.test_id attribute, value, or title. \<button> elements can also be found
-      # by their text content, and image \<input> elements by their alt attribute
+      # This can be any `<input>` element of type submit, reset, image, button or it can be a
+      # `<button>` element. All buttons can be found by their id, name, {Capybara.configure test_id} attribute, value, or title.
+      # `<button>` elements can also be found by their text content, and image `<input>` elements by their alt attribute.
       #
       # @overload find_button([locator], **options)
-      #   @param [String] locator            id, name, Capybara.test_id attribute, value, title, text content, alt of image
-      #
-      #   @overload find_button(**options)
```

Comment:
[note] I think the `@overload` is just a mistake. The current `#find_button` doc has no overloads: ⬇️ 

https://www.rubydoc.info/gems/capybara/3.21.0/Capybara/Node/Finders#find_button-instance_method

Reply:
agreed - but I don't think removing the `locator ` description makes sense